index.html

```diff
@@ -4116,91 +4036,328 @@ <h3>Parsing content</h3>
+          This also includes Denial of Service attacks against a solution
+          deployed with NFC tags (e.g. a malicious actor destroying tags linked
+          to a solution).
+        </li>
+        <li>
+          <strong>User identity or other privacy sensitive attributes</strong>
+          that can be directly or indirectly determined by using Web NFC, by the
+          <a>NFC content</a> creator by a web site using Web NFC.
+          This data could be used directly or leaked forward to third parties. Examples are user location, device identifiers and user identifiers.
+        </li>
+        <li>
+          <strong>User data</strong> exposed to a web page using Web NFC.
+          While web page might collect user data using other means than
+          Web NFC, it might embed this data into NDEF records and share via
+          Web NFC.
+        </li>
+        <li>
+          <strong>Integrity of user device</strong>.
+          A read of an NFC tag might result in a user device compromise
+          that can further lead to loss of other web NFC or platform assets.
+        </li>
+      </ul>
+    </div>
   </section>
 
-  <section> <h3>Permissions and user prompts</h3>
-  <p>
-    This specification attempts to minimize user prompting and uses implicit
-    security policies to address the
-    <a href="http://w3c.github.io/web-nfc/security-privacy.html#threats-and-possible-solutions">
-    threats</a>.
-    However, this specification does not describe, nor does it
-    mandate specific user prompting policies. The term <a>obtain permission</a>
-    is used for acquiring trust for a given operation.
-  </p>
-  <p class="note">
-    The [[[PERMISSIONS]]] API is suggested to be used by
-    UAs for implementing NFC related [[[PERMISSIONS]]] in order to minimize
-    the need for user prompting.
-  </p>
-  <p>
-    All <a>expressed permission</a>s that are preserved beyond the current
-    browsing session MUST be revocable.
-  </p>
+  <section class="informative"> <h2>Attacker model</h2>
+    <p>
+      The following attacker patterns have been considered:
+      <ul>
+        <li>
+          <strong>Malicious web page creator</strong>: phishing user data,
+          identity or other privacy sensitive attributes, destroying or
+          modifying NFC tags to cause further damage through fake identities
+          and attack vectors.
+        </li>
+        <li>
+          <strong>Malicious NFC tag creator</strong>: same as malicious web page
+          owner, but it has a possibility to create, delete or modify the NFC
+          tags locally.
+          As a result can compromise integrity of user device, cause data
+          injection, redirect to malicious web page, phishing user location,
+          causing side actions such as installing applications, trigger
+          automated dispatching or other actions.
+        </li>
+        <li>
+          <strong>Malicious MitM user</strong>: any MitM (man in the middle)
```

Comment:
Dont people write this all uppercase MITM?

Reply:
Right. I replaced `MitM` with `man-in-the-middle` and (`MITM` in one occurrence), according to the spelling in https://en.wikipedia.org/wiki/Man-in-the-middle_attack